code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -47,27 +47,27 @@
 
 /obj/machinery/chem_master/attackby(var/obj/item/weapon/B as obj, var/mob/user as mob)
 
-	if(istype(B, /obj/item/weapon/reagent_containers/glass))
+	if(istype(B, /obj/item/weapon/reagent_containers/glass) || istype(B, /obj/item/weapon/reagent_containers/food))
 
 		if(src.beaker)
-			user << "A beaker is already loaded into the machine."
+			user << "A \the [beaker] is already loaded into the machine."

```

Comment:
Instead of this, which will print "A the glass beaker ...", I recommend `"\A [beaker] ..."`